easybuild/easyconfigs/x/xalt/xalt-2.7.29.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'EB_XALT'
+
+name = 'xalt'
+version = '2.7.29'
+
+homepage = 'https://github.com/xalt/xalt'
+description = """XALT 2 is a tool to allow a site to track user executables
+  and library usage on a cluster. When installed it can tell a site what are
+  the top executables by Node-Hours or by the number of users or the number
+  of times it is run. XALT 2 also tracks library usage as well. XALT 2 can
+  also track package use by R, MATLAB or Python. It tracks both MPI and
+  non-MPI programs."""
+
+toolchain = SYSTEM
+
+# The location of XALT configuration file must be specified via
+# 'config_py' easyconfig parameter.  You can edit the easyconfig file,
+# or use 'eb --try-amend=config_py=<path>'. " See
+# https://xalt.readthedocs.io/en/latest/030_site_filtering.html for
+# more information.
+config_py = None
+
+# The name of the system must be specified via the 'syshost'
+# easyconfig parameter.  You can edit the easyconfig file, or use 'eb
+# --try-amend=syshost=<string>'.  See
+# https://xalt.readthedocs.io/en/latest/020_site_configuration.html
+# for more information.
+syshost = None
+
+# The XALT transmission method must be specified via the
+# 'transmission' easyconfig parameter.  You can edit the easyconfig
+# file, or use 'eb --try-amend=transmission=<string>'.  See
+# https://xalt.readthedocs.io/en/latest/020_site_configuration.html
+# for more information.
+transmission = None
+
+# Additional options
+executable_tracking = True
+gpu_tracking = None
+logging_url = None
+mysql = False
+scalar_sampling = True
+
+source_urls = ['https://github.com/xalt/xalt/archive/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['9c93d9077919532bd40638c4608be129a130cd80265334efa74db91afff5a2f4']
+
+builddependencies = [
+    ('cURL', '7.60.0-GCCcore-7.3.0'),
+    ('flex', '2.6.4-GCCcore-7.3.0'),
+    ('util-linux', '2.32.1-GCCcore-7.3.0'),
+]
+
+osdependencies = [
+    ('bc'),
+    ('coreutils'),
+    ('libelf-dev', 'elfutils-libelf-devel'),
+    ('libssl-dev', 'openssl-devel'),
```

Comment:
For this, we have a dedicated constant now: `OS_PKG_OPENSSL_DEV` (see https://github.com/easybuilders/easybuild-framework/pull/3309).

That avoids introducing many variants of this across easyconfigs. Here, `libopenssl-devel` is missing as an alternative for example, so using `OS_PKG_OPENSSL_DEV` instead fixes that.

You'll need EasyBuild v4.2.1 (or current `develop` version) for this to work.